Makefile

```diff
@@ -79,20 +79,36 @@ test-setup: ## setup test environment
 	test -f ${ENVTEST_ASSETS_DIR}/setup-envtest.sh || curl -sSLo ${ENVTEST_ASSETS_DIR}/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/v0.8.3/hack/setup-envtest.sh
 	source ${ENVTEST_ASSETS_DIR}/setup-envtest.sh; fetch_envtest_tools $(ENVTEST_ASSETS_DIR)
 
-integration-suite: ## Provision and run integration tests with cleanup
-	make integration-setup && \
-	make integration-run && \
-	make integration-cleanup
+kind-integration-suite: ## Provision and run integration tests with cleanup
```

Comment:
workflow task names needs to be updated.
https://github.com/aws/aws-cloud-map-mcs-controller-for-k8s/blob/main/.github/workflows/integration-test.yml#L33